nova/gateway.py

```diff
@@ -127,3 +128,8 @@ def __init__(
 
     async def close(self):
         return await self._api_client.close()
+    
+    def _host_with_prefix(host: str) -> str:
```

Comment:
```suggestion
    def _host_with_prefix(host: str) -> str:
       """
       Examples:
       >>> _host_with_prefix("https://example.com")
       https://example.com
       >>> _host_with_prefix("example.com")
       http://example.com
       """
```

maybe add some doctest to the function like this above